Vagrantfile

```diff
@@ -194,14 +206,17 @@ Vagrant.configure("2") do |config|
     },
     ubuntu: {
       name: 'ubuntu/bionic64',
-      version: '20190514.0.0'
+      version: '20190514.0.0',
+      scripts: [
+        {
+          name: "install-python",
```

Comment:
Use single-quote (since it's what we are using above): https://github.com/rubocop-hq/ruby-style-guide#consistent-string-literals

(would really love to run `rubocop` on this file in the CI, but would need a tailored config since it's Ruby used as a DSL).

Reply:
Gah it's inconsistent in this file, I was matching what's in the other `vm.provision` declarations